### PR TITLE
🌐 [i18n-KO] Translated `perplexity.mdx` to Korean

### PR DESCRIPTION
**Pull request by HanNayeoniee**
# What does this PR do?

Translated `perplexity.mdx` file of the documentation to Korean.
Added draft of `model_summary.mdx` file because it's referenced.
Thank you in advance for your review. 

Part of https://github.com/huggingface/transformers/issues/20179

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [x] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

Team PseudoLab, may you please review this PR? @0525hhgus, @KIHOON71, @sim-so, @gabrielwithappy, @HanNayeoniee, @wonhyeongseo, @jungnerd

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)

@sgugger, @ArthurZucker, @eunseojo May you please review this PR?

### CODE REVIEW

File: docs/source/ko/perplexity.mdx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+중요한 점은 토큰화 과정이 모델의 펄플렉서티에 직접적인 영향을 미치므로 서로 다른 모델을 비교할 때 항상 이를 고려해야 합니다.
+
+이는 데이터와 모델 예측 간의 cross-entropy 값에 지수를 취한 것과 동일합니다.
+펄플렉서티와 문자당 비트 수(BPC) 및 데이터 압축과의 관계에 대해 더 직관적인 이해를 원하신다면 다음 글
+[fantastic blog post on The Gradient](https://thegradient.pub/understanding-evaluation-metrics-for-language-models/)을 확인하세요.
+
+## 고정 길이 모델의 펄플렉서티(PPL) 계산하기[[calculating-ppl-with-fixedlength-models]]
+
+모델의 컨텍스트 크기가 정해져있지 않다면,
+아래와 같이 시퀀스를 자동 회귀적으로 분해하고 각 단계에서 이전 전체 시퀀스를 조건부에 넣어 모델의 펄플렉서티를 계산할 것입니다.
```
Comment (KIHOON71) [inline docs/source/ko/perplexity.mdx:37]:
```suggestion
아래와 같이 시퀀스를 자동 회귀적으로 분해하고 각 단계에서 선행 하는 전체 시퀀스를 조건부 확률에 넣어 모델의 펄플렉서티를 계산할 것입니다.
```
이렇게 하면 좀 더 뜻이 명확해질 것 같은데 어떻게 생각하시나요..?

Comment (HanNayeoniee) [reply docs/source/ko/perplexity.mdx]: 제안해주신대로 수정했습니다!

File: docs/source/ko/perplexity.mdx [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# 고정 길이 모델의 펄플렉서티(Perplexity)[[perplexity-of-fixedlength-models]]
+
+[[open-in-colab]]
+
+펄플렉서티(Perplexity, PPL)는 가장 일반적인 언어 모델 평가지표 중 하나입니다.
+자세히 알아보기 전에 이 평가지표는 고전적인 언어 모델(자기회귀 또는 인과적 언어 모델이라고도 함)에만 적용되며 BERT와 같은 마스킹된 언어 모델에는 잘 적용하지 않습니다 (BERT는 [summary of the models](model_summary) 문서를 참고하세요).
```
Comment (0525hhgus) [inline docs/source/ko/perplexity.mdx:18]:
```suggestion
자세히 알아보기 전에 이 평가지표는 고전적인 언어 모델(자기회귀 또는 인과적 언어 모델이라고도 함)에만 적용되며 BERT와 같은 마스킹된 언어 모델에는 잘 적용하지 않습니다 (BERT는 [모델 요약](model_summary) 문서를 참고하세요).
```
`summary of the models`을 `모델 요약`으로 번역했습니다!

File: docs/source/ko/perplexity.mdx [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+그러나 모델의 근사치를 구할 때는 일반적으로 모델이 처리할 수 있는 토큰 수에 제한이 있습니다.
+예를 들어, 가장 큰 버전의 [GPT-2](model_doc/gpt2)는 토큰의 길이가 1024로 고정되어 있습니다.
+따라서 \\(t\\) 가 1024보다 큰 경우에 \\(p_\theta(x_t|x_{<t})\\) 을 계산할 수 없습니다.
+
+대신 시퀀스는 일반적으로 모델의 최대 입력 크기와 동일한 길이는 가지는 부분 시퀀스로 쪼갭니다.
+만약 모델의 최대 입력 길이가 \\(k\\) 라면, 
+토큰 \\(x_t\\) 의 우도 값을 계산할 때 이전 토큰을 모두 사용하지 않고, \\(k-1\\) 토큰까지 사용해 대략적인 우도 값을 추정합니다. 
+
+모델의 시퀀스에 대한 펄플렉서티를 계산할 때,
+수월하지만 차선책은 시퀀스를 청크로 쪼개고 분해된 각 부분의 로그 우도 값을 독립적으로 합산하는 것입니다.
```
Comment (0525hhgus) [inline docs/source/ko/perplexity.mdx:50]:
```suggestion
수월하지만 차선책은 시퀀스를 묶음(chunk)으로 쪼개고 분해된 각 부분의 로그 우도 값을 독립적으로 합산하는 것입니다.
```
Glossary에서 묶음으로 되어 있어서 수정했는데, 문맥상 어울리지 않는 것 같기도 해서 참고만 해주세요!

File: docs/source/ko/perplexity.mdx [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+
+ppl = torch.exp(torch.stack(nlls).mean())
+```
+
+스트라이드를 최대 입력 길이와 동일하게 설정하면 위에서 설명한 차선책인 비슬라이딩 윈도우 전략과 동일합니다.
+일반적으로 스트라이드가 작을수록 모델이 각 예측을 할 때 더 많은 컨텍스트를 볼 수 있게 되어 펄플렉서티 값이 좋아집니다.
+
+위의 계산을 토큰이 겹치지 않도록 `stride = 1024`로 설정하면 PPL은 `19.44`로 GPT-2 논문에서 보고된 `19.93`과 거의 동일합니다.
+`stride = 512`로 슬라이딩 윈도우 전략을 사용하면 PPL은 `16.45`로 떨어집니다. 
+이는 더 좋은 점수일 뿐만 아니라 시퀀스 확률의 실제 자동 회귀 분해에 더 가까운 방식으로 계산됩니다.
```
Comment (0525hhgus) [inline docs/source/ko/perplexity.mdx:136]:
```suggestion
이는 더 좋은 점수일 뿐만 아니라 시퀀스 우도의 실제 자동 회귀 분해에 더 가까운 방식으로 계산됩니다.
```
원문에서 `likelihood`로 되어 있어서 우도로 수정했습니다!

**Review by 0525hhgus — CHANGES_REQUESTED** [review]
좋은 번역 감사합니다 👍 참고한 문서까지 꼼꼼하게 추가해주셨네요!
아래 리뷰 코멘트 남깁니다 😄

**Comment by sim-so** [comment]
좋은 번역 감사합니다! 앞서 올려주신 수정 제안 이외에 추가 의견 없습니다!

File: docs/source/ko/perplexity.mdx (hunk 3 above)
Comment (HanNayeoniee) [reply docs/source/ko/perplexity.mdx:50]: 기존 glossary를 따라가는게 통일성 있을 것 같아서, 묶음으로 수정했습니다!

File: docs/source/ko/perplexity.mdx (hunk 2 above)
Comment (HanNayeoniee) [reply docs/source/ko/perplexity.mdx]: 참고하는 문서 제목이 [The Transformer model family](https://huggingface.co/docs/transformers/model_summary)로 변경되어서, `Transformer 모델군`으로 번역했습니다.

File: docs/source/ko/perplexity.mdx [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+    input_ids = encodings.input_ids[:, begin_loc:end_loc].to(device)
+    target_ids = input_ids.clone()
+    target_ids[:, :-trg_len] = -100
+
+    with torch.no_grad():
+        outputs = model(input_ids, labels=target_ids)
+
+        # loss is calculated using CrossEntropyLoss which averages over valid labels
+        # N.B. the model only calculates loss over trg_len - 1 labels, because it internally shifts the labels
+        # to the left by 1.
```
Comment (ArthurZucker) [inline docs/source/ko/perplexity.mdx:119]: A nit but this could also be in Korean! 😉

File: docs/source/ko/model_summary.mdx
Comment (ArthurZucker) [inline docs/source/ko/model_summary.mdx:1]: We can maybe just leave it empty or add a comment saying that this is a draft!

**Review by ArthurZucker — APPROVED** [review]
Sorry for the late review! 
Nice addition 😉

File: docs/source/ko/model_summary.mdx
Comment (wonhyeongseo) [reply docs/source/ko/model_summary.mdx:1]: @HanNayeoniee May you please try referencing it as `../en/model_summary`? Let's see how it turns up on the live-preview. If it works, we can simply filter out `../en/` later on when we finish translating referenced documents.

Thank you so much for your review, @ArthurZucker ! 😄

Comment (ArthurZucker) [reply docs/source/ko/model_summary.mdx:1]: Ah right I forgot that the default was kept in english my bad!

File: docs/source/ko/perplexity.mdx (hunk 5 above)
Comment (HanNayeoniee) [reply docs/source/ko/perplexity.mdx]: Thanks for your reveiw! I updated L117-118.

File: docs/source/ko/model_summary.mdx
Comment (HanNayeoniee) [reply docs/source/ko/model_summary.mdx:1]: My apologies for the late reply.
Instead of adding a korean draft file, I referenced english one  as `../en/model_summary` and deleted `model_summary`.
`../ko/model_summary` would be merged in here #24625

**Comment by HanNayeoniee** [comment]
@sgugger, @ArthurZucker, @eunseojo May you please review this PR?

**Review by ArthurZucker — APPROVED** [review]
Thanks a lot for updating 🤗
